code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -82,20 +82,23 @@
 	set src in view(1)
 
 	if(isghost(usr))
-		to_chat(usr, "You ghost!")
+		to_chat(usr, "You can't do this as a ghost.")
```

Comment:
Soulless.

Reply:
tbh this should be just a return with no text but who cares